AppCenterDataStorage/AppCenterDataStorage/Internal/Ingestion/MSStorageIngestion.h

```diff
@@ -0,0 +1,33 @@
+#import <Foundation/Foundation.h>
+#import "MSHttpIngestion.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * The header name for update token.
+ */
+static NSString *const kMSHeaderUpdateApiToken = @"x-api-token";
```

Comment:
The naming looks like the same then the header for the update api token in distribute. Should probably rename to your use case. What is this token used for in storage, "update token" is only for distribute?

Reply:
changed to "app-secret"